src/frontend/next/src/pages/_document.tsx

```diff
@@ -0,0 +1,50 @@
+import { Children } from 'react';
+import Document, { Html, Head, Main, NextScript, DocumentContext } from 'next/document';
+import { ServerStyleSheets } from '@material-ui/core/styles';
+
+// Reference: https://github.com/mui-org/material-ui/blob/master/examples/nextjs/pages/_document.js
+class MyDocument extends Document {
+  static async getInitialProps(ctx: DocumentContext) {
+    const initialProps = await Document.getInitialProps(ctx);
+    return { ...initialProps };
+  }
+
+  render() {
+    return (
+      <Html lang="en">
+        <Head>
+          <link
+            rel="stylesheet"
+            href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
```

Comment:
Let's not do the font this way.  In Gatsby we pull in the font via npm, so we should do the same here, and load it locally out of our app vs. using third-party CDN.

Reply:
@humphd : removed them.